### DISCUSSION TITLE
EvalAI Submission General Questions

### DISCUSSION THREAD

**Discussion by jasoon-chen** [empriselab/RCareWorld#171] [Help - Programming] [answered]
I submitted a zip folder onto eval.AI however do you know if there needs a be a manual review of the submission folder or is there an automatic system check that is supposed to display my score? Currently my result file is just none. Just want to confirm that my submission successfully submitted through. I submitted around Nov 10, 2024 5:33:30 PM Pacific Timezone. Thanks!

![image](https://github.com/user-attachments/assets/5338b261-6024-4072-9909-150b58928c8f)

**Comment by YoruCathy** [comment, marked as answer]
Hi, there needs to be a manual run of it. We'll update the leader board soon.

**Reply by shxingch** [reply]
Hi @jasoon-chen ! We manually run your submission file and you can now see your score on the leaderboard.

**Reply by jasoon-chen** [reply]
Thanks I just saw the submission. Is it possible to get some more detailed breakdown on the evaluation? I seem to be getting differing results in terms of score when I run my solution locally compared to the score provided.

Edit: Is this being done with the randomized positions of the objects and humans? If so, is it possible to replicate this onto the environment?

**Reply by shxingch** [reply]
We manually checked and ran your submission file in a new cloned environment. I will send the running log file from the run to your email. You can check if there were any issues in the uploaded docker file.
